pages/data-providers/index.jsx

```diff
@@ -47,17 +47,21 @@ const useDataProviderController = () => {
   return [formRef, dataProvider, resetDataProvider]
 }
 
+const searchUrlFor = (id) => `https://core.ac.uk/search?q=repositories.id:${id}`
+
+const SUPPORT_EMAIL_URL = 'mailto:t%68%65t%65am%40core%2e%61c%2eu%6b'
+const SUPPORT_EMAIL = decodeURIComponent(
+  SUPPORT_EMAIL_URL.slice('mailto:'.length)
+)
+
 const getHelperMessage = ({ created, duplicated, error }) => {
   if (error) {
     return {
       message: (
         <>
-          We could detect neither repository nor journal. Please, provide the
-          exact OAI-PMH endpoint URL. If you are having trouble contact us at{' '}
-          <Link href="mailto:thet&#101;&#97;m&#64;c&#111;re&#46;&#97;c&#46;&#117;k">
-            thet&#101;&#97;m&#64;c&#111;re&#46;&#97;c&#46;&#117;k
-          </Link>
-          .
+          We cannot detect a repository or a journal. Please, provide the exact
+          OAI-PMH endpoint. If you are having trouble contact us at{' '}
+          <Link href={SUPPORT_EMAIL_URL}>{SUPPORT_EMAIL}</Link>.
```

Comment:
We cannot detect a repository or a journal **at this address.** 
Please, provide the exact OAI-PMH endpoint URL **(do we have a link to an help page somewhere?)** . If you are having trouble contact us at{' '}

Reply:
I asked @samuelpearce to provide a link. We have a [help page](https://kmi-ou.atlassian.net/servicedesk/customer/portal/2/topic/a5556556-aff6-4a2f-ab14-f48f9d557f53/article/4227093) but he said it's not ready yet, so I removed it.

Feel free to use 'Insert a suggestion' feature, it's a plus/minus button on the toolbar.

Reply:
For _at this address_, I wanted it to be short. If you find it not obvious or better with more words, I will add it.